packages/browser-integration-tests/suites/replay/customEvents/test.ts

```diff
@@ -117,7 +117,7 @@ sentryTest(
             nodeId: expect.any(Number),
             node: {
               attributes: {
-                'aria-label': '** ***** ** **********',
+                'aria-label': 'An Error in aria-label',
```

Comment:
Intentional, we are no longer masking this by default

Reply:
I remember we talked about this, but can't remember anymore in detail - why are we not masking this by default anymore? 🤔